testsuite/tests/typing-abstract-kinds/basics.ml

```diff
@@ -0,0 +1,766 @@
+(* TEST
```

Comment:
I'd like to see one or two simple tests that `module type of` does the right thing wrt kinds.